src/core_plugins/metric_vis/public/metric_vis.js

```diff
@@ -38,6 +38,7 @@ function MetricVisProvider(Private) {
         name: 'metric',
         title: 'Metric',
         min: 1,
+        aggFilter: ['!derivative'],
```

Comment:
its already excluded as tile map defines which aggs to use (incontrary to here where which aggs not to use are defined)